sources/packages/api/src/services/application/application.service.ts

```diff
@@ -578,12 +578,9 @@ export class ApplicationService extends RecordDataModelService<Application> {
     if (
       sortField &&
       sortField !== "status" &&
-      databaseFieldOfApplicationDataTable(sortField)
+      sortApplicationsSummary(sortField)
```

Comment:
It seems that it is still targeting the UI. Is it not supposed to be something like _sortApplicationsColumnMap_?